test/Microsoft.AspNetCore.SignalR.Client.Tests/HttpConnectionTests.cs

```diff
@@ -39,8 +39,8 @@ public void ConnectionReturnsUrlUsedToStartTheConnection()
         }
 
         [Theory]
-        [InlineData((HttpTransportType)0)]
-        [InlineData(HttpTransportType.All + 1)]
+        [InlineData(HttpTransportType.None)]
+        [InlineData((HttpTransportType)int.MaxValue)]
```

Comment:
I don't think this should be invalid. I know it was before, but I don't think it makes sense. In general, having all bits set in a flags enum isn't a problem. Neither do we need to check for negative numbers (in `DefaultTransportFactory` constructor) since this is a bit flag, not a number.